components/membership/membership-client/src/main/kotlin/net/corda/membership/client/MGMOpsClient.kt

```diff
@@ -13,8 +14,10 @@ interface MGMOpsClient : Lifecycle {
     /**
      * Generates the Group Policy file to be used to register a new member to the MGM
      *
-     * @param holdingIdentityShortHash The ID of the holding identity to be checked.
+     * @param holdingIdentityShortHash The holding identity ID of the MGM of the membership group to be joined.
```

Comment:
```suggestion
     * @param holdingIdentityShortHash The holding identity ID of the MGM.
```
I'm not sure tat "to be joined" makes sense for the generate group policy endpoint